subprojects/gradle/module-dependencies-graph/src/main/kotlin/com/avito/module/internal/dependencies/ModuleProjectDependenciesNode.kt

```diff
@@ -0,0 +1,21 @@
+package com.avito.module.internal.dependencies
+
+import com.avito.module.internal.configurations.ConfigurationCoordinate
+import org.gradle.api.Project
+
+internal class ModuleProjectDependenciesNode(
```

Comment:
In general, we only duplicate project structure. The benefits are not clear.
If we make it serializable, we could reuse this model in Worker API.

Reply:
The current benefit is having a small object with two fields
When we decide to move that module to the WorkerApi then will discuss
I can't see blockers for future refactoring if it will exist

Reply:
Ok, just wanted to sync probable options for reusing.